### PR TITLE
Use centralized method of sale channel to export inventory

### PR DESCRIPTION
**Pull request by tarunbhardwaj**

### CODE REVIEW

File: product.py [hunk 1]
```diff
@@ -229,6 +231,45 @@ def create_from(cls, channel, product_data):
         listing.save()
         return listing
 
+    def export_inventory(self):
+        """
+        Export inventory of this listing
+        """
+        Location = Pool().get('stock.location')
+
+        channel, product = self.channel, self.product
```
Comment (sharoonthomas) [inline product.py]: after the if ?

File: product.py [hunk 2]
```diff
@@ -229,6 +231,45 @@ def create_from(cls, channel, product_data):
...
+
+        if self.channel.source != 'magento':
+            super(ProductSaleChannelListing, self).export_inventory()
+
+        locations = map(int, Location.search([('type', '=', 'storage')]))
```
Comment (sharoonthomas) [inline product.py]: inventory associated with the warehouse of the channel ?

File: product.py [hunk 3]
```diff
@@ -229,6 +231,45 @@ def create_from(cls, channel, product_data):
         listing.save()
         return listing
```
Comment (sharoonthomas) [inline product.py]: add a class method `export_inventory_bulk`

Comment (tarunbhardwaj) [inline product.py]: `bulk inventory` is implemented in 

https://github.com/fulfilio/trytond-sale-channel/blob/edd366ee36f9cd9a70566acc74532573454bc972/product.py#L99-L109

when we will have bulk update API in magento then that can be implemented here.

**Pull request merged**